oai_agents/common/overcooked_dataset_et_eyetracking.py

```diff
@@ -1,25 +1,30 @@
-from oai_agents.common.arguments import get_arguments
 from oai_agents.common.state_encodings import ENCODING_SCHEMES
-from oai_agents.common.subtasks import Subtasks, calculate_completed_subtask
-from overcooked_ai_py.mdp.overcooked_env import OvercookedEnv
 from overcooked_ai_py.mdp.overcooked_mdp import OvercookedState, OvercookedGridworld, Action
+from sklearn.preprocessing import OneHotEncoder
+
 
 
 import json
-import numpy as np
 import pandas as pd
-from torch.utils.data import Dataset, DataLoader
-from tqdm import tqdm
-import sys
+from torch.utils.data import Dataset
+import numpy as np
+
+import warnings
+# Filter out UserWarnings
+warnings.simplefilter(action='ignore', category=UserWarning)
+
 
 class OvercookedDataset(Dataset):
-    def __init__(self, dataset, layouts, args, add_subtask_info=True):
-        self.seq_len = 15
+    def __init__(self, dataset, layouts, args, seq_len, num_classes, add_subtask_info=True):
+
+        self.seq_len = seq_len
+        self.num_classes = num_classes
         self.step = 1
         self.add_subtask_info = add_subtask_info
         self.encoding_fn = ENCODING_SCHEMES['OAI_feats']
-        self.data_path = '/content/HAHA/data/2019_hh_trials_all.pickle' #- For now the base classifier is based on 2019_hh_trials_all.pickle
+        self.data_path = '/Users/nikhilhulle/Desktop/HAHA-eyetracking/data/2019_hh_trials_all.pickle' #- For now the base classifier is based on 2019_hh_trials_all.pickle
```

Comment:
use a relative path within the repo or use some parameter since this wont work on anyone else's machine